extra-container/buildah/autobuild/build

```diff
@@ -0,0 +1,8 @@
+abinfo "Fetching source..."
+git clone https://github.com/containers/buildah
+cd "$SRCDIR"/buildah
+git checkout "tags/v$PKGVER"
+
+abinfo "Building buildah..."
+make PREFIX=/usr
+make DESTDIR="$PKGDIR" PREFIX=usr install install.completions
```

Comment:
You could have easily used the `plainmake` ABTYPE here. Leave `install.completions` in `beyond`.